src/components/container/container.js

```diff
@@ -251,9 +251,13 @@ export default class Container extends UIObject {
     this.trigger(Events.CONTAINER_CONTEXTMENU, this, this.name)
   }
 
-  setCurrentTime(time) {
-    this.trigger(Events.CONTAINER_SEEK, time, this.name)
-    this.playback.seek(time)
+  setCurrentTime(time, inseconds) {
```

Comment:
maybe we can assume `false` as default (so we won't broken anything) but I also think that the default should be **seconds** =/
```javascript
setCurrentTime(time, inseconds=false)
```